modin/pandas/series.py

```diff
@@ -500,6 +500,10 @@ def values(self):  # noqa: RT01, D200
 
         data = self.to_numpy()
         if isinstance(self.dtype, pd.CategoricalDtype):
+            from modin.config import ExperimentalNumPyAPI
+
+            if ExperimentalNumPyAPI.get():
+                data = data._to_numpy()
```

Comment:
We already converted `self` to a numpy array and assigned it to `data` var. It doesn't have `._to_numpy()` method.

Reply:
If the `ExperimentalNumPyAPI` flag is `True`, then we convert self to a modin.numpy.array - it seems that in this case, we're defaulting to `pandas` to give us an array, so we need to convert the modin.numpy.array object to an actual numpy.array object.